scripts/update/Updater.php

```diff
@@ -526,6 +526,14 @@ public function update($initialVersion)
         }
 
         $this->skip('12.23.0', '12.25.0');
+
+        if ($this->isVersion('12.25.0')) {
+            $widgetDefinitionsFilePath =  __DIR__ . '/../../core/ontology/widgetdefinitions.rdf';
+            $api = core_kernel_impl_ApiModelOO::singleton();
+            $api->importXmlRdf('http://www.tao.lu/datatypes/WidgetDefinitions.rdf', $widgetDefinitionsFilePath);
```

Comment:
```suggestion
            $core_kernel_impl_ApiModelOO::singleton()->importXmlRdf(
                WidgetRdf::NAMESPACE, 
                __DIR__ . '/../../core/ontology/widgetdefinitions.rdf'
            );
```

Sorry for annoying, but it seems we can avoid usage variables at all and improve a bit WidgetRdf interface, adding the new constant NAMESPACE and having something like that:

```
interface WidgetRdf
{
    public const NAMESPACE = 'http://www.tao.lu/datatypes/WidgetDefinitions.rdf';
    public const CLASS_URI_WIDGET = self::NAMESPACE . '#WidgetClass';
//    ...more code here
}
```

Reply:
No annoyance caused at all, your suggestion makes perfect sense, thank you!